### PR TITLE
Infrastructure agent 1.19.0 release notes

### PR DESCRIPTION
**Pull request by rubenruizdegauna**

### CODE REVIEW

File: src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1190.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+subject: Infrastructure agent
+releaseDate: '2021-06-29'
+version: 1.19.0
+---
+
+## Notes
+A new version of the agent has been released. Follow standard procedures to [update the Infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent).
+New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months.
```
Comment (x8a) [inline src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1190.mdx:9]: I'd rephrase the sentence to: `We recommend you to upgrade the agent every 3 months.`

**Review by x8a — COMMENTED** [review]
I only have a small comment, but it looks great. Thanks!

**Comment by rubenruizdegauna** [comment]
> Hi, @rubenruizdegauna! I only have a small comment, but it looks great. Thanks!

Changed! Thanks @x8a  :)

**Review by x8a — APPROVED** [review]
